pkgs/development/python-modules/svgutils/default.nix

```diff
@@ -25,13 +24,24 @@ buildPythonPackage rec {
 
   dependencies = [ lxml ];
 
-  doCheck = pythonOlder "3.12";
+  patches = [
+    # Remove nose dependency, see: https://github.com/btel/svg_utils/pull/131
 
-  nativeCheckInputs = [
-    pytestCheckHook
-    nose
+    # this first commit is required, as isort moved nose imports
+    (fetchpatch2 {
+      url = "https://github.com/btel/svg_utils/commit/48b078a729aeb6b1160142ab65157474c95a61b6.patch?full_index=1";
+      hash = "sha256-9toOFfNkgGF3TvM340vYOTkuSEHBeiyBRSGqqobfiqI=";
+    })
+
+    # migrate to pytest
```

Comment:
The commit message says “migrate to nose”, you might want to fix that :) A link to the PR would be good here too so that people know this hasn’t been merged upstream yet.